safe_transaction_service/history/views.py

```diff
@@ -243,7 +243,7 @@ def list(self, request, *args, **kwargs):
             return self.get_paginated_response([])
 
         all_tx_hashes = [element["safe_tx_hash"] for element in page]
```

Comment:
I haven't strong opinion about this, but maybe `all_safe_tx_hashes` or `all_tx_identifiers`?

Reply:
Done and also added a comment